app/src/main/kotlin/no/nav/k9punsj/felles/Typer.kt

```diff
@@ -48,3 +50,48 @@ data class Sak(
     enum class FagsakSystem { K9 }
 
 }
+
+enum class Søknadstype(
+    internal val k9YtelseType: String,
+    internal val brevkode: Brevkode,
+    internal val journalpostType: String) {
+    PleiepengerSyktBarn("PSB", Brevkode.PLEIEPENGER_BARN_SOKNAD, "PleiepengerSyktBarn"),
+    PleiepengerLivetsSluttfase("PPN", Brevkode.SØKNAD_PLEIEPENGER_LIVETS_SLUTTFASE, "PleiepengerLivetsSluttfase"),
+    Omsorgspenger("OMP", Brevkode.SØKNAD_UTBETALING_OMS, "UtbetaleOmsorgspenger"),
+    OmsorgspengerUtbetaling_Arbeidstaker("OMP", Brevkode.SØKNAD_UTBETALING_OMS_AT, "UtbetaleOmsorgspenger"),
+    OmsorgspengerUtbetaling_Papirsøknad_Arbeidstaker("OMP", Brevkode.PAPIRSØKNAD_UTBETALING_OMS_AT, "UtbetaleOmsorgspenger"),
+    OmsorgspengerUtbetaling_Korrigering("OMP", Brevkode.FRAVÆRSKORRIGERING_IM_OMS, "UtbetaleOmsorgspenger"),
```

Comment:
Er journalpostType `UtbetaleOmsorgspenger ` for korrigering av IM riktig her?

Reply:
Ja, disse blir brukt til å opprette gosys oppgaver. Men ser ikke ut som de har vart i bruk på ett tag så jag fjerner dem så får vi se vad som sker :D